mix.exs

```diff
@@ -26,7 +26,7 @@ defmodule Peridiod.MixProject do
   defp deps do
     [
       {:extty, "~> 0.2"},
-      {:peridio_rat, github: "peridio/peridio-rat", branch: "main"},
+      {:peridio_rat, github: "peridio/peridio-rat", branch: "file-write-sync"},
```

Comment:
Just double checking this is what we want.

Reply:
Nope, changed to main and updated to latest